index.html

```diff
@@ -0,0 +1,280 @@
+                        <path d="M21 8H1" stroke="#9C69E2" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"/>
+                    </svg>
+                </button>
+            </header>
+            <main class="main-content">
+                <section class="opportunity">
+                    <h1 class="opportunity__title title-color">Save your data storage here.</h1>
+                    <p class="opportunity__text text text-color">
+                        Data Warehouse is a data storage area that has been
+                        tested for security, so you can store your data here
+                        safely but not be afraid of being stolen by others.
+                    </p>
+                    <button class="button text button-color-purple" type="button">Learn more</button>
+                </section>
+                <section class="identity">
+                    <div class="identity__img">
+                        <img class="identity__img-size" src="images/second-screen.png" alt="second screen">
+                    </div>
+                    <div class="identity__content">
+                        <h2 class="identity__title title title-color">We are a high-level data storage bank</h2>
+                        <p class="text text-color">
+                            The place to store various data that you can access at any time through the internet
+                            and where you can carry it.
+                        </p>
+                        <p class="text text-color">
+                            This very flexible storage area has a high level of security. To enter into your own
+                            data you must enter the password that you created when you registered in this Data Warehouse. 
+                        </p>
+                    </div>
+                </section>
+                <section class="features">
+                    <div class="heading">
+                        <h2 class="heading__title title title-color">Features</h2>
+                        <p class="heading__text text text-color">
+                            Some of the features and advantages that we provide for those of you who store data
+                            in this Data Warehouse.
+                        </p>
+                    </div>
+                    <div class="cards">
+                        <div class="card">
+                            <img src="images/card-image.png" alt="card image" class="card__img" loading="lazy">
+                            <div class="card__content bg-one">
+                                <h3 class="card__title title-color">Search Data</h3>
+                                <p class="card__text interactive text-color">
+                                    Don’t worry if your data is very large, the Data Warehoue provides a search engine,
```

Comment:
The character U+2019 "’" could be confused with the character U+0060 "`", which is more common in source code. [Adjust settings]